Create sdv.sh

pk-mdt opened the pull request:
Create cron script to made set data value changes on a regular basis.

pk-mdt committed:
Create sdv.sh

Geczy commented:
Maybe something more robust to lv the value first and only change if it is different?

pk-mdt committed:
Update sdv.sh

pk-mdt commented:
Fixed the script so it successfully checks the value before making the change.
Added some additional checks to make it easier to modify for a particular sdv parameter.
